src/Colopl/Spanner/Concerns/ManagesStaleReads.php

```diff
@@ -68,7 +70,11 @@ public function selectWithTimestampBound($query, $bindings = [], TimestampBoundI
      */
     public function selectOneWithTimestampBound($query, $bindings = [], TimestampBoundInterface $timestampBound = null): ?array
     {
-        return $this->cursorWithTimestampBound($query, $bindings, $timestampBound)->current();
+        $result = $this->sessionNotFoundWrapper(function () use ($query, $bindings, $timestampBound) {
+			return $this->cursorWithTimestampBound($query, $bindings, $timestampBound)->current();
+		});
+        assert(is_null($result) || is_array($result));
```

Comment:
Why is there an assertion here?

Reply:
to mute phpstan error
```
 ------ ------------------------------------------------------------------------------------------------------------- 
  Line   Colopl/Spanner/Concerns/ManagesStaleReads.php (in context of class Colopl\Spanner\Connection)                
 ------ ------------------------------------------------------------------------------------------------------------- 
  73     Method Colopl\Spanner\Connection::selectOneWithTimestampBound() should return array|null but returns mixed.  
 ------ ------------------------------------------------------------------------------------------------------------- 
```

Reply:
Would adding `/** @var array<mixed>|null $result */` resolve the error as well?
I would like to avoid running asertions when we can get the same results without hindering performance.

Reply:
This is why I used assert(), it has special feature in recent php - any code inside it doesn't run in production mode
from https://www.php.net/manual/en/ini.core.php#ini.zend.assertions
>When set to -1, assertion code will not be generated, making the assertions zero-cost (production mode).
but I will do as you prefer

Reply:
Yeah. I just read up on this. Sorry for the confusion. This can stay as is.